cockatrice/src/messagelogwidget.cpp

```diff
@@ -325,33 +325,33 @@ void MessageLogWidget::logMoveCard(Player *player,
     if (targetZoneName == tableConstant()) {
         soundEngine->playSound("play_card");
         if (card->getFaceDown()) {
-            finalStr = tr("%1 puts %2 into play%3 face down.");
+            finalStr = tr("%1 puts %2 into play %3 face down.");
         } else {
-            finalStr = tr("%1 puts %2 into play%3.");
+            finalStr = tr("%1 puts %2 into play %3.");
```

Comment:
Now that I'm reading this again... this two strings are the only ones that have a different placement.

I leave it to the native speakers to tell me if the wording is correct or should be similar to the others:
 `finalStr = tr("%1 puts %2 %3 into play.");`

<br>

"`Player 1` puts `Shock` `from their hand` into play."
over
"`Player 1` puts `Shock` into play `from their hand."`

Reply:
While both options are technically correct, the second option ("`Player 1` puts `Shock` into play `from their hand`") sounds much more natural in English, to me at least.

Reply:
More native speakers opinions, please. :)

If `2)` is the best choice, we should change the other strings accordingly as well. 👍

Reply:
I'm good with either choice, but I do like 2 better